plugins/content/local/writer.go

```diff
@@ -138,6 +138,14 @@ func (w *writer) Commit(ctx context.Context, size int64, expected digest.Digest,
 	if err := os.Rename(ingest, target); err != nil {
 		return err
 	}
+	f, err := os.Open(target)
```

Comment:
rename is atomic. I think you should sync parent dir instead of new file name.

```
Calling fsync() does not necessarily ensure that the entry in the
       directory containing the file has also reached disk.  For that an
       explicit fsync() on a file descriptor for the directory is also
       needed.
```

https://man7.org/linux/man-pages/man2/fsync.2.html

Reply:
done, thank you @fuweid